ntp-proto/src/peer.rs

```diff
@@ -96,12 +96,21 @@ pub enum IgnoreReason {
     TooOld,
 }
 
+#[allow(dead_code)]
```

Comment:
What is dead here that requires this? Perhaps we could limit this to a single field, which also helps identify when we can remove it better.